gqlalchemy/query_builder.py

```diff
@@ -372,28 +372,33 @@ class DeclarativeBase(ABC):
     def __init__(self, connection: Optional[Union[Connection, Memgraph]] = None):
         self._query: List[Any] = []
         self._connection = connection if connection is not None else Memgraph()
+        self._clauses: List[DeclarativeBaseTypes] = []
```

Comment:
Are we planning some extensive use of this list of clauses? It seems to me that as of now it is being read from only to check if there is "RETURN" statement. If we have some plans around this list, then ok, otherwise, i would replace it with simple boolean